gitlab/v4/objects/keys.py

```diff
@@ -0,0 +1,26 @@
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import GetMixin
+
+__all__ = [
+    "Key",
+    "KeyManager",
+]
+
+
+class Key(RESTObject):
+    pass
+
+
+class KeyManager(GetMixin, RESTManager):
+    _path = "/keys"
+    _obj_cls = Key
+
+    def get(self, id=None, **kwargs):
+        if id is not None:
+            return super(KeyManager, self).get(id, **kwargs)
+
+        if "fingerprint" not in kwargs:
+            raise AttributeError("Missing attribute: fingerprint")
```

Comment:
This should probably have both options:

```suggestion
            raise AttributeError("Missing attribute: id or fingerprint")
```

Reply:
Done.